packages/custom-props-layer/src/types.ts

```diff
@@ -12,6 +12,7 @@ export type TabBodyProps = {
   openColorPicker: (
     colorPickerProps: Omit<ColorPickerDialogProps, "onCrossClick">
   ) => void;
+  options?: string[];
```

Comment:
Add a comment here about which prop types need options. Currently, only the enum prop type needs options.

Reply:
Done